common/structs.go

```diff
@@ -0,0 +1,6 @@
+package common
+
+// Repository has the fields required for syncing
+type Repository struct {
+	Slug, Description, Source, Target string
+}
```

Comment:
I think if you run 'make format' it will use gofmt to auto-format all of your files.  It should fix things like missing newlines, and probably re-write things like line 5 here to be separate lines.